lib/jira/commands/tickets.rb

```diff
@@ -0,0 +1,22 @@
+module Jira
+  class CLI < Thor
+
+    desc "tickets", "List the tickets of the input username"
```

Comment:
According to https://docs.atlassian.com/jira/REST/latest/#api/2/search-search, that command defaults maxResults to 50 so at most only 50 results should be returned. Definitely solve this issue later.